mentors/python/e.json

```
@@ -1 +1,8 @@
-[]
+[{
+  "github_username": "EricKolibacz",
+  "name": "Eric Kolibacz",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "Python was the first programming language I got engaged with, back in the days when I was still in high school. Exersicm's mentoring program is a great oppurtunity to give somehting back to community while improving my own coding skill."
+}]
```

```suggestion
  }
]
```
Should mirror the top of the file essentially.